### PR TITLE
Updates middlware and site course validators to use multiple edx-orgs

### PR DESCRIPTION
**Pull request by rehan99000**

### CODE REVIEW

**Review by asadmanzoor93 — CHANGES_REQUESTED** [review]
Add/update related unit tests.

**Comment by rehan99000** [comment]
> Add/update related unit tests.

The change does not affect any tests and is just to adjust the changes in https://github.com/edly-io/edx-platform/pull/253

File: ecommerce/extensions/edly_ecommerce_app/helpers.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def encode_edly_user_info_cookie(cookie_data):
     return jwt.encode(cookie_data, settings.EDLY_COOKIE_SECRET_KEY, algorithm=settings.EDLY_JWT_ALGORITHM).decode('utf-8')
 
 
-def get_edx_org_from_edly_cookie(encoded_cookie_data):
+def get_edx_orgs_from_edly_cookie(encoded_cookie_data):
     """
     Returns "edx-org" value from the "edly-user-info" cookie.
```
Comment (taimoor-ahmed-1) [inline ecommerce/extensions/edly_ecommerce_app/helpers.py:44]: "edx-org" -> "edx-orgs"

Comment (rehan99000) [reply ecommerce/extensions/edly_ecommerce_app/helpers.py]: fixed

**Review by taimoor-ahmed-1 — APPROVED** [review]
👍

**Review by asadmanzoor93 — APPROVED** [review]

**Pull request merged**